pgconfeu-recap.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Recap of PGConf.EU 2024 in Athens'
+description: 'My notes from PGConf.EU 2024 in Athens, Greece. Talks, extension summit, diversity committee meeting, Xata Postgres dinner, and more!'
+image:
+  src: https://us-east-1.storage.xata.sh/uq2d57g5hsgdo2ih1nnau2kdbpi6l05c
+  alt: 'PGConfEU 2024 in Athens'
+author: Gulcin Yildirim Jelinek
+authorEmail: gulcin@xata.io
+date: 11-15-2024
+tags:
+  [
+    'postgres',
+    'pgconfeu',
+    'pgeu',
+    'athens',
+    'xata',
+    'postgresql',
+    'postgres-dinner',
+    'postgres-europe',
+    'europe',
+    'extensions',
+    'extension-development',
+    'extension-summit',
+    'diversity-task-force',
+    'diversity-committee',
+    'pgroll',
+    'pgzx',
+    'zig',
+    'schema-migration',
+    'zero-downtime',
+    'zero-downtime-deployment',
+    'zero-downtime-schema-changes',
+    'zero-downtime-migrations',
+    'instant-rollback',
+    'rollback',
+    'uptime',
+    'pgconf',
+    'pgconf-eu',
+    'pgconf-europe',
+    'pgconf-2024',
+    'pgconf-2024-athens',
+    'cloudnativepg',
+    'kubernetes',
+    'p2d2',
+    'prague',
+    'fosdem'
+  ]
+published: true
+slug: pgconfeu-recap-2024
+ogImage: https://us-east-1.storage.xata.sh/uq2d575fqbbtbk8h135drrsm2u27lu88
+---
+
+## PGConf.EU 2024 Recap
+
+We finished yet another PGConf.EU in Athens, Greece at the end of October.
+PGConf.EU is a very special conference for me, and I've written a blog post about it previously [here](https://xata.io/blog/pgconfeu-2024).
+
+Now that the conference is over, I'd like to share some more details about the conference, my experiences, and the talks I attended.
+
+## CloudNativePG Training
+
+Day O of PGConf.EU is when we do training sessions.
+My dear friends and former colleagues, Gabriele Bartolini and Leonardo Cecchi, held a [CloudNativePG training session](https://www.postgresql.eu/events/pgconfeu2024/schedule/session/5998-mastering-postgresql-in-kubernetes-with-cloudnativepg/).
```

Comment:
Not necessary, but for all the speaker names it's an opportunity to link to their LinkedIn or Twitter profiles